### ISSUE TITLE
[java] UnusedImports: False positive if nested interface is referenced

### ISSUE THREAD

**Issue by eekboom** [pmd/pmd#2001]
**Affects PMD Version:** 
6.17.0
**Rule:**
category/java/bestpractices.xml/UnusedImports
**Description:**
When a class A is imported and the only reference to it is in javadoc {@link A.B} where B is a nested interface, then the warning triggers incorrectly.

**Code Sample demonstrating the issue:**

```
package de.eekboom.acme.util;

public class A {
	public interface B {}
}
```

```
package de.eekboom.acme;

import de.eekboom.acme.util.A;

/**
 * Foobedoo {@link A.B}
 */
public class C {
}
```

**Running PMD through:** Gradle

**Comment by jsotuyod** [comment]
This is no longer reproducible under PMD 7.0.0
